app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -572,6 +576,93 @@ export default class SampleViewV2 extends React.Component {
     return {};
   };
 
+  countReportRows = () => {
+    const { filteredReportData, reportData } = this.state;
+    let total = reportData.length;
+    let filtered = filteredReportData.length;
+    reportData.forEach(genusRow => {
+      total += genusRow.species.length;
+      filtered += genusRow.filteredSpecies.length;
+    });
+    return { total, filtered };
+  };
+
+  filteredMessage = () => {
+    const { total, filtered } = this.countReportRows();
+    return (
+      filtered != total &&
+      `${filtered} rows passing the above filters, out of ${total} total rows.`
```

Comment:
The filtered rows message persists after clicking `Clear All Filters` and shows that not all rows are passing, despite no filters being selected.

![image](https://user-images.githubusercontent.com/53838890/69571236-71525480-0f76-11ea-9e76-f1384101ced4.png)

Reply:
Fixed